test/snippet/utility/type_pack/detail/type_pack_algorithm_all_of.cpp

```diff
@@ -17,7 +17,7 @@ auto fn = [](auto value)
     else if constexpr (std::is_same_v<value_t, int>)
         return value == 3;
     else if constexpr (std::is_same_v<value_t, double>)
-        return std::abs(value - 1.2) < 0.00001;
+        return value - 1.2 < 0.00001;
```

Comment:
it has never been constexpr or it is not constexpr in only libc++?

since it is only a sample snippet I guess the semantic change does not matter.

Reply:
`constexpr` in C++23. It's been `constexpr` for a long time in libstdc++, but isn't in `libc++`. I could check whether there is a feature macro and whatnot, but since this snippet is just for illustration, and the `abs` doesn't matter in the snippet, I might as well just remove it.